helm/cluster-operator/templates/deployment.yaml

```diff
@@ -1,18 +1,18 @@
 apiVersion: extensions/v1beta1
 kind: Deployment
 metadata:
-  name: cluster-operator
-  namespace: {{ .Values.namespace }}
+  name: {{ tpl .Values.resource.default.name  . }}
+  namespace: {{ tpl .Values.resource.default.namespace  . }}
   labels:
-    app: cluster-operator
+    app: {{ tpl .Values.resource.default.name  . }}
 spec:
   replicas: 1
   strategy:
     type: Recreate
   template:
     metadata:
       labels:
-        app: cluster-operator
+        app: {{ tpl .Values.resource.default.name  . }}
```

Comment:
That should be split into app and version like here https://github.com/giantswarm/aws-operator/blob/542f4938d2b30573b904c609eb1bfcd4f7a6860c/helm/aws-operator/templates/deployment.yaml#L7-L8

Reply:
Done.